cli/azd/pkg/tools/bicep/bicep.go

```diff
@@ -93,33 +88,33 @@ func (cli *bicepCli) CheckInstalled(ctx context.Context) (bool, error) {
 }
 
 func (cli *bicepCli) Build(ctx context.Context, file string) (string, error) {
-	sniffCliVersion := func() (string, error) {
-		verRes, err := cli.runCommand(ctx, "version", "--out", "json")
-		if err != nil {
-			return "", fmt.Errorf("failing running az version: %s (%w)", verRes.String(), err)
-		}
+	// sniffCliVersion := func() (string, error) {
+	// 	verRes, err := cli.runCommand(ctx, "--version", "--out", "json")
+	// 	if err != nil {
+	// 		return "", fmt.Errorf("failing running az version: %s (%w)", verRes.String(), err)
+	// 	}
 
-		var jsonVer struct {
-			AzureCli string `json:"azure-cli"`
-		}
+	// 	var jsonVer struct {
+	// 		AzureCli string `json:"azure-cli"`
+	// 	}
 
-		if err := json.Unmarshal([]byte(verRes.Stdout), &jsonVer); err != nil {
-			return "", fmt.Errorf("parsing cli version json: %s: %w", verRes.Stdout, err)
-		}
+	// 	if err := json.Unmarshal([]byte(verRes.Stdout), &jsonVer); err != nil {
+	// 		return "", fmt.Errorf("parsing cli version json: %s: %w", verRes.Stdout, err)
+	// 	}
 
-		return jsonVer.AzureCli, nil
-	}
+	// 	return jsonVer.AzureCli, nil
+	// }
```

Comment:
Do we still need this check? If i recall the history of this issue it only errors out when called through `az` but the standalone worked correctly??  @ellismg ?